src/chrome/DropdownMenu.tsx

```diff
@@ -58,7 +65,7 @@ const DropdownMenu: React.FC<DropdownMenuProps> = ({
         role="menu"
         aria-orientation="vertical"
         aria-labelledby="options-menu"
-        style={{ minWidth: '9rem' }}
+        style={{ minWidth: '9rem', ...oneItemStyles}}
```

Comment:
no need for the `const` declaration above, just use the ternary operator directly:

```ts
style={oneItem
  ? { minWidth: '9rem',  top: -8, right : 30 }
  : { minWidth: '9rem' }
}

I can't remember how the linter likes multiline ternary operators, so please do whatever the linter says is right 😄